app/models/rent.rb

```diff
@@ -185,8 +163,8 @@ def edit?(user)
   # Returns the length of the booking in hours, as an virtual attribute
   # /d.wessman
   def duration
-    if self.d_from.present? && self.d_til.present?
-      h = (self.d_til-self.d_from)/3600
+    if d_from.present? && d_til.present?
+      h = (d_til-d_from)/3600
```

Comment:
Surrounding space missing for operator '/'.<br>Surrounding space missing for operator '-'.